resources/metadata/whitesource.yaml

```diff
@@ -372,16 +372,16 @@ spec:
       - STAGES
       - STEPS
       default: -1
-    - name: includes
+    - name: include
       type: string
-      description: Space separated list of file path patterns to include in the scan, slashes must be escaped for sed
+      description: Space separated list of file path patterns to include in the scan
       mandatory: false
       scope:
         - PARAMETERS
         - STAGES
         - STEPS
-      default: '**\/src\/main\/**\/*.java **\/*.py **\/*.go **\/*.js **\/*.ts'
-    - name: excludes
+      default: '**/src/main/**/*.java **/*.py **/*.go **/*.js **/*.ts'
+    - name: exclude
```

Comment:
shouldn't we stick with `excludes`? This is in line with stashing on Jenkins

Reply:
I was trying to stay consistent with Fortify's `exclude` flag.

Reply:
see above